services/src/test/resources/add-thumb-it-servlet.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2008 The University of North Carolina at Chapel Hill
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:mvc="http://www.springframework.org/schema/mvc"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xsi:schemaLocation="
+        http://www.springframework.org/schema/beans
+        http://www.springframework.org/schema/beans/spring-beans.xsd
+        http://www.springframework.org/schema/context
+        http://www.springframework.org/schema/context/spring-context-3.0.xsd
+        http://www.springframework.org/schema/mvc
+        http://www.springframework.org/schema/mvc/spring-mvc-3.0.xsd">
+
+    <mvc:annotation-driven/>
+
+    <context:component-scan resource-pattern="**/EditThumbnailController*" base-package="edu.unc.lib.dl.cdr.services.rest"/>
+
+    <bean id="messageSender" class="org.mockito.Mockito" factory-method="mock">
+        <constructor-arg value="edu.unc.lib.dl.services.MessageSender"/>
+    </bean>
+
+    <bean id="importThumbnailService" class="edu.unc.lib.dl.cdr.services.processing.ImportThumbnailService">
+        <property name="aclService" ref="aclServices" />
+        <property name="messageSender" ref="messageSender" />
+        <property name="dataDir" value="dataDir" />
+    </bean>
+
+    <bean id="aclServices" class="org.mockito.Mockito" factory-method="mock">
+        <constructor-arg value="edu.unc.lib.dl.acl.fcrepo4.AccessControlServiceImpl"/>
+    </bean>
+
+    <bean id="dataDir" class="java.lang.String">
+        <constructor-arg value="src/test/resources/temp"/>
```

Comment:
use the `target` directory for temp space, its used for throw away and build stuff that shouldn't get checked into git.